content/docs/cli.md

```diff
@@ -206,10 +219,16 @@ This will install new instance of Kafka Operator which maps to Kafka operator ve
 
 ### Install a Package Overriding Instance Name and Parameters
 
-Use `--instance` and `--parameter`/`-p` for setting an instance name and parameters, respectively:
+Use `--instance` for setting an instance name.
+Use `--parameter`/`-p` and/or `--parameter-file`/`-P` for setting parameters.
+
+For example:
 
 ```bash
-$ kubectl kudo install kafka --instance=my-kafka-name --parameter ZOOKEEPER_URI=zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181 --parameter ZOOKEEPER_PATH=/small -p BROKERS_COUNTER=3
+$ cat kafka-parameters.yaml
+ZOOKEEPER_URI: zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181
+ZOOKEEPER_PATH: /small
+$ kubectl kudo install kafka --instance=my-kafka-name -p BROKERS_COUNTER=3 -P kafka-parameters.yaml
```

Comment:
Might as well fix the parameter name for the Kafka operator while you're editing this line - it should be `BROKER_COUNT`